lib/app/dataMixin.js

```diff
@@ -80,5 +80,12 @@ export default {
         this.$route.path
       )
     }
+  },
+  created () {
```

Comment:
It shouldn't be placed here, or it will be invoked for each component.

Reply:
If we don't put it here, we have some other choices:
1. Ask users to put it mannually into the custom `Layout.vue`
2. Wrap the `themeLayout` with another components, to which inject this snippets.